lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>.write(f.read()) je trochu prasárna, ale tak máme malej soubor, tak ti to odpustim, snad se účastníci nebudou inspirovat :-)</p><p> </p><p>použil bych shutil.copyfileobj, případně tady se přímo nabízí urllib.request.urlretrieve.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Díky za urlretrieve.</p>